spin.toml

```diff
@@ -1,36 +1,40 @@
-spin_manifest_version = "1"
-authors = ["karthik2804 <karthik.ganeshram@fermyon.com>"]
-description = "A spin app to explore the the sqlite DB"
+spin_manifest_version = 2
+
+[application]
 name = "spin-sqlite-web-cli"
-trigger = { type = "http", base = "/" }
 version = "0.1.0"
+description = "A spin app to explore the the sqlite DB"
+authors = ["karthik2804 <karthik.ganeshram@fermyon.com>"]
+
+[application.trigger.http]
+base = "/"
 
 [variables]
 sqlite_username = { required = true }
 sqlite_password = { required = true }
 
-[[component]]
-id = "sql-cli"
+[[trigger.http]]
+id = "trigger-sql-cli"
+component = "sql-cli"
+route = "/internal/sqlite/..."
+
+[[trigger.http]]
+id = "trigger-dev"
+component = "dev"
+route = "/static/..."
+
+[component.sql-cli]
 source = "target/spin-sqlite-web-cli.wasm"
+variables = { sqlite_username = "{{ sqlite_username }}", sqlite_password = "{{ sqlite_password }}" }
 exclude_files = ["**/node_modules"]
+allowed_outbound_hosts = ["redis://*:*", "mysql://*:*", "postgres://*:*"]
 sqlite_databases = ["default"]
-[component.trigger]
-route = "/internal/sqlite/..."
-[component.build]
+
+[component.sql-cli.build]
 command = "npm run build"
-watch = [
-    "assets/sqlite-cli.html",
-    "assets/index.js",
-    "assets/index.css",
-    "src/**/*",
-]
-[component.config]
-sqlite_username = "{{ sqlite_username }}"
-sqlite_password = "{{ sqlite_password }}"
-
-[[component]]
+watch = ["assets/sqlite-cli.html", "assets/index.js", "assets/index.css", "src/**/*"]
+
+[component.dev]
 source = { url = "https://github.com/fermyon/spin-fileserver/releases/download/v0.0.3/spin_static_fs.wasm", digest = "sha256:38bf971900228222f7f6b2ccee5051f399adca58d71692cdfdea98997965fd0d" }
-id = "dev"
 files = [{ source = "assets", destination = "/" }]
-[component.trigger]
-route = "/static/..."
+allowed_outbound_hosts = ["redis://*:*", "mysql://*:*", "postgres://*:*"]
```

Comment:
Definitely no reason for the fileserver to be accessing any of these!

Reply:
Ah these come from using `spin doctor` to migrate to `v2`. Will remove.